roles/pulp_common/defaults/main.yml

```diff
@@ -46,7 +46,7 @@ pulp_upgraded_manually: false
 # keep this two in sync with pulp_repos/defaults/main.yml
 # due to CI coverage test
 __pulp_pkg_repo_name: "pulpcore"
-pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.16/el{{ ansible_distribution_major_version }}/$basearch/"
+pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.17/el{{ ansible_distribution_major_version }}/$basearch/"
```

Comment:
@mikedep333 I'm seeing this on a Fedora 35 (source) vagrant box

```
Errors during downloading metadata for repository 'pulpcore':
  - Status code: 404 for https://yum.theforeman.org/pulpcore/3.17/el35/x86_64/repodata/repomd.xml (IP: 151.101.2.49)
Error: Failed to download metadata for repo 'pulpcore': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
test                                                                                                                                                                                                           38 MB/s |  86 MB     00:02    
Ignoring repositories: pulpcore
```

Note the "el35", it's treating the fedora version as a centos version